tests/CMakeLists.txt

```diff
@@ -1,3 +1,37 @@
+# Thanks to Daniel Blezek <blezek@gmail.com> for the GTEST_ADD_TESTS code
```

Comment:
You can reference where you copied it from - CMake's FindGTest - instead of the author of this function that is already cited there